src/tests/intg/test_enumeration.py

```diff
@@ -85,8 +85,8 @@ def cleanup_ldap_entries(ldap_conn, ent_list=None):
     """Remove LDAP entries added by create_ldap_entries"""
     if ent_list is None:
         for ou in ("Users", "Groups", "Netgroups", "Services", "Policies"):
-            for entry in ldap_conn.search_s("ou=" + ou + "," +
-                                            ldap_conn.ds_inst.base_dn,
+            for entry in ldap_conn.search_s("ou=%s,%s" % (ou,
+                                            ldap_conn.ds_inst.base_dn),
```

Comment:
Can you use the new `f-Strings` feature to concatenate those strings? I think it will be easier to read than your proposal.

Reply:
done

Reply:
Did you forget to push the changes? I can still see the previous version.

Reply:
I was checking if there are more occurrences before pushing.
The best way of addressing this would be to move the function to a library instead as it is duplicated all around, but for now this should do.